RIOT-OS/RIOT
# tests/hacl: Move from unittests to regular test

### Contribution description

another one related to #10187

### Testing procedure

Verify that `tests/pkg_hacl` works: `BOARD=<your-favourite> make -C tests/pkg_hacl flash test`

### Issues/PRs references

#10187
